### ISSUE TITLE
Lint errors not recognized on version 0.10

### ISSUE THREAD

**Issue by nicomem** [DoozyX/clang-format-lint-action#19]
On the given C++ file:
```cpp
#include <iostream>

int main() {
    std::cout
      <<
      "This is a test"
      <<
      std::endl
      ;

    return EXIT_SUCCESS;
}
```

And with the following github action configuration:
```yml
# [...]
jobs:
  myjob:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2

    - name: clang-format lint
      uses: DoozyX/clang-format-lint-action@v0.10
      with:
        source: './src/test.cpp'
        style: llvm
```

The clang-format does not report any error, even though it should.

This problem does not happen in the v0.9 version.
